cmd/cloud-controller-manager/app/core.go

```diff
@@ -74,15 +74,15 @@ func startCloudNodeLifecycleController(ctx context.Context, controllerContext ge
 	)
 	if err != nil {
 		klog.Warningf("failed to start cloud node lifecycle controller: %s", err)
-		return nil, false, nil
```

Comment:
Was this intended to return `nil` before?

Reply:
yes